packages/opentelemetry-cloud-monitoring-exporter/test/monitoring.test.ts

```diff
@@ -69,7 +71,7 @@ describe('MetricExporter', () => {
 
       metricDescriptors = sinon.spy(
         /* tslint:disable no-any */
-        (request: any, callback: (err: Error | null) => void): any => {
+        (request: any, params: any, callback: (err: Error | null) => void): any => {
```

Comment:
See spied on code on line 187 in `packages/opentelemetry-cloud-monitoring-exporter/src/monitoring.ts`